<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSetDlg</class>
 <widget class="QWidget" name="CSetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSetDlg</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>591</width>
     <height>571</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="set_base">
    <attribute name="title">
     <string>基本设置</string>
    </attribute>
    <widget class="QGroupBox" name="m_log_groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>104</y>
       <width>581</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>日志信息设置</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>是否生成日志：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="m_cblog">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>27</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>日志路径：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_logpath">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>27</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="m_set_logpath">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>26</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="m_checkup_groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>178</y>
       <width>581</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>检查文件存放路径</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>检查结果存放路径：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>53</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>图片文件存放路径：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_patient_path">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>25</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_pdf_path">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>54</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="m_set_patient_path">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>23</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_set_pdf_path">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>52</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>269</y>
       <width>581</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>FTP信息</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>IP：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>304</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>端口：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>304</x>
        <y>48</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_ftp_ip">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_ftp_port">
      <property name="geometry">
       <rect>
        <x>349</x>
        <y>20</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_ftp_user">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_ftp_psw">
      <property name="geometry">
       <rect>
        <x>349</x>
        <y>50</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>453</y>
       <width>581</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>数据源信息</string>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>19</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>DSN：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>51</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_dsn">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_db_user">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>51</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_db_psw">
      <property name="geometry">
       <rect>
        <x>349</x>
        <y>51</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>13</y>
       <width>581</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>基本信息</string>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>体检号长度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>48</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>BreasTouch 版本：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>扫码枪是否带回车：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_id_len">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="m_have_enter">
      <property name="geometry">
       <rect>
        <x>418</x>
        <y>20</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF文件上传方式：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="m_pdf_upload">
      <property name="geometry">
       <rect>
        <x>419</x>
        <y>49</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="m_bt_version">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>49</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>581</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>share 信息</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>19</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>IP：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>51</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>文件夹名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_share_ip">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_share_folder">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="field_info">
    <attribute name="title">
     <string>数据库信息</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>581</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>存放患者表字段信息</string>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>表或试图名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_tablename">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>21</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>234</x>
        <y>81</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>体检号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_patient_id">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>247</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_patient_name">
      <property name="geometry">
       <rect>
        <x>281</x>
        <y>49</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>411</x>
        <y>48</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>年龄：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_patient_age">
      <property name="geometry">
       <rect>
        <x>442</x>
        <y>49</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>411</x>
        <y>79</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_patient_sex">
      <property name="geometry">
       <rect>
        <x>442</x>
        <y>80</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>52</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>男性存储方式：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="m_man_store">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>50</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>82</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>年龄是否需要计算：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="m_age_convert">
      <property name="geometry">
       <rect>
        <x>113</x>
        <y>80</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>581</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>数据库</string>
     </property>
     <widget class="QLineEdit" name="m_db_code">
      <property name="geometry">
       <rect>
        <x>84</x>
        <y>23</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>数据库编码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>21</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>数据库类型:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="m_db_type">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="m_bok">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>592</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_bcancel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>592</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
